src/modules/i18n/LanguageAsyncStorage.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+
+import { AsyncStorage } from 'react-native'
+
+const LANGUAGE_KEY = 'CONTENT_LANGUAGE'
+
+class LanguageAsyncStorage {
+  asyncStorage: AsyncStorage
+
+  constructor (asyncStorage: AsyncStorage = AsyncStorage) {
+    this.asyncStorage = asyncStorage
+  }
+
+  loadLanguage = (): Promise<?string> => {
+    return this.asyncStorage.getItem(LANGUAGE_KEY)
+  }
+
+  setLanguage = async (language: string) => {
+    await this.asyncStorage.setItem(LANGUAGE_KEY, language)
+  }
+}
```

Comment:
We might add `contentLanguage` (as well as `currentCity` later) to the SettingsAsyncStorage to access all stuff in the async storage the same way :thinking:

Reply:
I thought about that as well, but imo it's clearer to split it. That way, everything that gets displayed in the settings belongs to AppSettings whereas localization stuff has its separate place (I'll rename the LanguageAsyncStorage to Localization)

Reply:
I talked with @maxammann about this at the conference and we agreed that it's fine. But I'm also not really sure.

Reply:
This could interfere with settings:

https://github.com/Integreat/integreat-react-native-app/blob/46edaca973393a32fc52a626341d50e9299b3d30/src/modules/settings/AppSettings.js#L46

I think it is better to put the settings into a `settings` sub-object.

Reply:
Isn't this an unnecessary overhead? There won't be more localization props than `contentLanguage` and `selectedCity`, and those won't be in the settings. So I would rather just  add a comment to mind the other AsyncStorage.

Reply:
As discusses in call: Move to the AppSettings